backend/de.metas.async/src/main/java/de/metas/lock/api/impl/Lock.java

```diff
@@ -85,18 +90,6 @@ public String toString()
 		return ObjectUtils.toString(this);
```

Comment:
offtopic: consider getting rid of ObjectUtils.toString().
Replace it with `@ToString` and `@ToString.Exclude`.

Reply:
done (note: didn't find anything that i wanted to exclude)